dev-resources/alignments/empty.json

```diff
@@ -1 +1 @@
-{}
+[]
```

Comment:
I added this file as kind of a hack.. the core lib should really consider alignments optional and default this value